docs/001_develop/02_server-capabilities/006_core-business-logic-event-handler/index.mdx

```diff
@@ -459,7 +459,48 @@ As with `onValidate` the block must finish with an [`ack`](#ack). If the event i
     }
   }
 
-``` 
+```
+
+##### `trigger`
+
+The `trigger` method can be used to call on another event within the same Event Handler script file. It takes the event metadata as a parameter.
```

Comment:
Is it definitely the same script file? or same event handler? 

If I have 5 event handler files running under the same service, do I still use triggerRemote?

Reply:
yes it has to be the same script file for trigger to work, any other script file you must use triggerRemote with the current implementation

Reply:
OK, I think that is fine, but we need to make that clearer in triggerRemote (which states different microservice not different file)

If we must have 2 separate calls (?) then I think its better per file because users can compact processes differently in dev to, say, a prod env and would need to change the code accordingly